client-httpurlconnection/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,2 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Let's add the license header here:
https://github.com/mozilla-mobile/android-components/blob/master/components/browser/domains/src/main/AndroidManifest.xml

Reply:
Done